subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -148,23 +148,50 @@ static void handle_wifi_disconnect_result(struct net_mgmt_event_callback *cb)
 	}
 }
 
+static void print_twt_params(uint8_t dialog_token, uint8_t flow_id,
```

Comment:
isn't it easier and scalable just to pass the structure instead of individual parameters?

Reply:
The twt setup command(wifi_twt_flow_info) and twt response (wifi_twt_params) does not have uniform strructure. In that case we have to create local structure of  wifi_twt_flow_info type,assign each parameter and then pass it to helper function. Is there any other way?

Reply:
Both setup and response use same structure `struct wifi_twt_params`, I guess you mean twt setup/response vs. flow?

Looking at the structures we can easily use `struct wifi_twt_flow_info` inside `struct wifi_twt_params`, and then have helpers to print out request/response and flow information?

Reply:
Yeah, but needs a lot of changes to existing code base(both zephyr and nrf). 
I have no problem in doing so, if it has to be that way, the requirement was to show the parameters.

Reply:
Its fine we can do the refactoring later if its too much trouble.